.github/workflows/docker-pr.yml

```diff
@@ -0,0 +1,27 @@
+name: Docker PR Build
+
+on:
+  pull_request:
+    branches: [development, main]
+
+jobs:
+  build_docker_image:
+    name: Build Docker image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
```

Comment:
Can we use 3.8 here as we do with all other versions and tests?

Reply:
I have updated the Version to `"3.8"` and the build is successful, please take a look

<img width="1470" alt="image" src="https://github.com/fossasia/eventyay-tickets/assets/75414859/913efb45-fcdf-4368-bf32-877d222d4cfa">